test/e2e/commands/network.test.mjs

```diff
@@ -103,15 +103,10 @@ describe('NetworkCommand', () => {
         flags.applicationProperties.constName,
         flags.bootstrapProperties.constName,
         flags.chainId.constName,
-        flags.deployHederaExplorer.constName,
-        flags.deployMirrorNode.constName,
```

Comment:
I think we can remove flags.deployMirrorNode from the flags file and the entire system, since it is implied when we call `solo mirror-node deploy`

Reply:
done